internal/commands/reminder.go

```diff
@@ -23,19 +23,33 @@ func requestStatus(ctx context.Context, client bot.Client, logger log.Logger, re
 		if err != nil {
 			logger.Error(
 				ctx,
-				"command/reminder.requestStatus GetIncident error",
-				log.NewValue("channelID", incident.ChannelId),
+				log.Trace(),
+				log.NewValue("reason", "GetIncident"),
+				log.NewValue("channelId", incident.ChannelId),
+				log.NewValue("channelName", incident.ChannelName),
 				log.NewValue("error", err),
 			)
 			return
 		}
 
+		logger.Info(
+			ctx,
+			log.Trace(),
+			log.NewValue("action", "running"),
+			log.NewValue("channelID", incident.ChannelId),
+			log.NewValue("channelName", incident.ChannelName),
+		)
+
 		if canStopReminder(incident) {
 			logger.Info(
 				ctx,
-				"command/reminder.requestStatus stop",
+				log.Trace(),
+				log.NewValue("action", "do_not_notify"),
+				log.NewValue("reason", "canStopReminder"),
```

Comment:
What do you think of extracting this to a function inside `log` package. Something like:
```
log.Action("do_not_notify")
log.Reason("canStopReminder")
```

Reply:
Done